### PR TITLE
APPS-6013: SyncAPI build code produces broken code

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon 

- Ticket: https://spryker.atlassian.net/browse/APPS-6013

- Release Group: https://release.spryker.com/release-groups/view/4612

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   SyncApi               | patch (currently 0.x)  |                       |

-----------------------------------------

### Improvements
- Fixed TransferName resolving.

### CODE REVIEW

File: infection.json5 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "source": {
```
Comment (dereuromark) [inline infection.json5]: Those files should not be in the root, but inside tests/ or some subdir IMO

**Comment by matweew** [comment]
the release group is wrong. https://release.spryker.com/release-groups/view/4622 is already released.

File: src/SprykerSdk/SyncApi/Message/SyncApiMessageFormatter.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * Info text will be in green, everything in double quotes will be yellow, and quotes will be removed.
+     *
+     * @param string $message
+     *
+     * @return string
+     */
+    public static function format(string $message): string
+    {
+        if (PHP_SAPI === 'cli' && strtolower(substr(PHP_OS, 0, 3)) !== 'win') {
```
Comment (matweew) [inline src/SprykerSdk/SyncApi/Message/SyncApiMessageFormatter.php]: Some windows terminal emulators already support coloring
```
if (getenv('TERM') === 'xterm-color' || getenv('TERM') === 'xterm-256color') {
    // The terminal supports color
} else {
    // The terminal does not support color
}
```

This checks if the value of TERM is either xterm-color or xterm-256color, which indicates that the terminal supports 8-color or 256-color support, respectively.

File: src/SprykerSdk/SyncApi/Message/SyncApiMessageFormatter.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Colorize output in CLI on Linux machines.
+     *
+     * Info text will be in green, everything in double quotes will be yellow, and quotes will be removed.
+     *
+     * @param string $message
+     *
+     * @return string
+     */
+    public static function format(string $message): string
```
Comment (matweew) [inline src/SprykerSdk/SyncApi/Message/SyncApiMessageFormatter.php:24]: I don't see that it's used in the PR expect tests

**Review by matweew — APPROVED** [review]

**Pull request merged**